doc/installation.rst

```diff
@@ -80,11 +80,13 @@ computer, you can install them in your home directory like this:
 ::
 
   cd <dir>
+  rm -rf build Build
```

Comment:
Without this, if we compiled it in the past, it can generate corrupted stuff. We get hit ourself by this and this is was reported by user.